src/lib.rs

```diff
@@ -152,12 +217,36 @@ impl<'a, W: Write> TurtleFormatter<'a, W> {
             node.child_by_field_name("label")
                 .map_or("", |n| n.utf8_text(self.file).unwrap_or(""))
         });
+        let tab_length = if self.options.align_prefix_iris {
```

Comment:
I am not a huge fan of this option. I would prefer not to have it except if you have good reasons.

Reply:
I just added this (prefix IRI aligning) because I am used to it, but there is no good reason, indeed. It is definitely bad for diff-optimization, so leaving it out is fine with me.